docs/osx/el_capitan_10.11.2_install_notes.md

```diff
@@ -0,0 +1,87 @@
+WebSiteOne install
+————————
+
+Fresh 10.11.2 (El Capitan) OSX
+
+1) Using git for the first time prompted installing developer tools
+
+2) pull down code
+
+```
+git clone http://github.com/AgileVentures/WebsiteOne
```

Comment:
yes, - although those are my rough notes from what I did personally which was work off the main repo, which when I'm project manager I always do.  

if these are polished up to be better notes then we should probably change that - but I wonder if we should even include this at all?  maybe I should put a :construction: at the top or something ...